spec/Vehicle/Vehicle.vspec

```diff
@@ -207,10 +205,8 @@
   description: Indicates whether the vehicle is stationary or moving
 
 - AverageSpeed:
-  datatype: int32
+  datatype: float
   type: sensor
-  min: -250
```

Comment:
The proposal is to remove all specific limits concerning speed in the VSPEC standard. The current limits (-250/+250) does not relate to any industry standard (?), and actual limits will be vehicle dependent. If limits are needed/wanted, which e.g. could be useful for "CruiseControl.SpeedSet" then they should be defined in the VSPEC instantiation for that particular vehicle.

Reply:
Agree, I see no need for the limits.

Reply:
I agree. Most min/max (e.g. speed) are deployment specific (might go into specific layer used for deployment, but not into the core speed)